Resources/public/js/models/ez-locationmodel.js

```diff
@@ -178,6 +178,66 @@ YUI.add('ez-locationmodel', function (Y) {
                 location.reset();
                 callback(error, response);
             });
+        },
+
+        /**
+         * Loads path of the current location. The result is the array containing
+         * eZ.Location objects present on the path. The array doesn't contain current location.
+         *
+         * @param {Object} options
+         * @param {Object} options.api the JS REST client instance
+         * @param {Function} callback
```

Comment:
you should document that the callback receives an error and the result as an array with something like:
```
@param {Function} callback
@param {false|Error} callback.error
@param {Array} callback.locations the array of Locations
```